### PR TITLE
feat: add dockerless installation guide

### LINKED ISSUE #107: Detailed native ("dockerless") installation guide
- install node
- install go
- install xud
- install connext
- install lnd's
- connect em (xud.conf, copying tls.certs et al)

### PR DESCRIPTION
**Pull request by kilrau**
Closes https://github.com/ExchangeUnion/docs/issues/107

Preview @ https://docs.exchangeunion.com/v/develop/development/dockerless-guide

### CODE REVIEW

File: Dockerless Installation.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+## Daemonize `tor`
+
+TODO
+
+
+# Putting it all together
+
+Create the following `xud.conf` in `/home/xud/.xud`:
+```json
```
Comment (michael1011) [inline Dockerless Installation.md]: This is not JSON but TOML

File: Dockerless Installation.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+[lnd.LTC]
+disable = true
+```
+
+For convenience, consider adding `alias xucli='/home/xud/xud/bin/xucli -p 8886'` to the xud user's `.bashrc` and source it.
+
+# Tips 'n Tricks
+
+* When installing on a Raspberry Pi you might see `Unexpected error during initialization`. [Here](https://github.com/ExchangeUnion/xud/issues/1199#issuecomment-527819108) the solution. If you see an `install error` when installing via `npm i`, try `npm install --production`.
```
Comment (michael1011) [inline Dockerless Installation.md]: When only installing the `--production` dependencies, you won't be able to compile the TypeScript

File: Dockerless Installation.md (hunk 1 above)
Comment (kilrau) [reply Dockerless Installation.md]: Changed.

Comment (kilrau) [reply Dockerless Installation.md]: Even though gitbook seems to only recognize json, not toml. Anyhow.

File: Dockerless Installation.md (hunk 2 above)
Comment (kilrau) [reply Dockerless Installation.md]: Added separate typescript installation.

**Comment by kilrau** [comment]
Give this another skim. Keep in mind that this is for the pro's and not supposed to be perfect and complete.

File: Dockerless Installation.md [hunk 3]
```diff
@@ -69,7 +69,7 @@ KillSignal=SIGINT
 
 # Connext
 
-TODO: install (waiting for vector)
+Follow the [docs](https://github.com/connext/vector#quick-start).
```
Comment (michael1011) [inline Dockerless Installation.md]: This is not good enough imho:

- there are multiple vector stacks
```
Four different Vector stacks are supported:

global: standalone messaging service (+ EVMs in dev-mode)
node: vector node + database
router: vector node + router + database
duet: 2x node/db pairs, used to test one-on-one node interactions
trio: 2x node/db pairs + 1x node/router/db , used to test node interactions via a routing node.
```
Which one do I need?

- also what kind of endpoints do I need to expose for xud?
- and what node should my vector node connect to for swapping with other xud vector clients?

Comment (ghost) [reply Dockerless Installation.md]: 1. Docker pull the vector_node image `docker pull connextproject/vector_node:837bafa1`

2. Create `node.config.json` for simnet:
```
{
  "adminToken": "ddrWR8TK8UMTyR",
  "chainAddresses": {
    "1337": {
      "channelFactoryAddress": "0x2eC39861B9Be41c20675a1b727983E3F3151C576",
      "channelMastercopyAddress": "0x7AcAcA3BC812Bcc0185Fa63faF7fE06504D7Fa70",
      "transferRegistryAddress": "0xB2b8A1d98bdD5e7A94B3798A13A94dEFFB1Fe709",
      "TestToken": ""
    }
  },
  "chainProviders": {
    "1337": "http://35.234.110.95:8545"
  },
  "domainName": "",
  "logLevel": "debug",
  "messagingUrl": "https://messaging.connext.network",
  "production": true,
  "mnemonic": "crazy angry east hood fiber awake leg knife entire excite output scheme"
}
```

3. Start node:
```
#!/bin/bash
set -xe
docker run --env VECTOR_CONFIG="$(cat node.config.json)" --env VECTOR_PROD=true --env VECTOR_SQLITE_FILE="/database/store.db" -p "8000:8000" --mount type=volume,source=vector_node_store,destination=/database --name vector_node --rm connextproject/vector_node:837bafa1
```

Comment (kilrau) [reply Dockerless Installation.md]: `docker pull` conflicts with a **dockerless** deployment guide. @erkarl 

Agree that it's not good enough, question is do we want to fix that now or as follow up when vector is out and stabl-ish. @michael1011

Comment (ghost) [reply Dockerless Installation.md]: In that case you want the `node: vector node + database` version.

Comment (kilrau) [reply Dockerless Installation.md]: Added that

**Comment by kilrau** [comment]
Good to merge? @michael1011 @erkarl

File: Dockerless Guide.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+Environment="LEGACY_MODE=true"
+WorkingDirectory=/home/xud/connext/
+ExecStart=node /home/xud/connext/build/src/index.js
+KillMode=process
+KillSignal=SIGINT
+```
+
+# Tor
+
+You can install tor via `sudo apt install tor` on most linux distros nowadays, just make sure [the version is fairly recent](https://github.com/torproject/tor/releases). If not, consult the [tor installation guides](https://2019.www.torproject.org/docs/installguide.html.en). Run `tor --version` to verify the tor process is running.
```
Comment (michael1011) [inline Dockerless Guide.md]: `tor --version` does *not* check whether the Tor process is running

File: Dockerless Guide.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+```
+
+For convenience, consider adding `alias xucli='/home/xud/xud/bin/xucli -p 8886'` to the xud user's `.bashrc` and source it. Then restart `xud` once (e.g. with `systemctl restart xud`) and try running `xucli getinfo`, which should return with an overview of xud's, as well as lnd's and connext status.
+
+Ping us in the help channel of our [Discord server](https://discord.gg/YgDhMSn) for support.
+
+# Tips 'n Tricks
+
+* When installing on a Raspberry Pi you might see `Unexpected error during initialization`. [Here](https://github.com/ExchangeUnion/xud/issues/1199#issuecomment-527819108) the solution.
+* If you see an `install error` when installing via `npm install`, try `npm install --production` & `npm install -g typescript`.
```
Comment (michael1011) [inline Dockerless Guide.md]:
```suggestion
* If you see an `install error` when installing via `npm install`, try `npm install --production` & `npm install typescript`.
```

TypeScript does not have to be installed globally. Just inside the package is enough

**Review by michael1011 — CHANGES_REQUESTED** [review]

File: Dockerless Guide.md (hunk 4 above)
Comment (kilrau) [reply Dockerless Guide.md]: Fixed

File: Dockerless Guide.md (hunk 5 above)
Comment (kilrau) [reply Dockerless Guide.md]: Fixed

**Review by michael1011 — APPROVED** [review]

**Pull request merged**